doc/docs/configuration.md

```diff
@@ -0,0 +1,20 @@
+# Configuration
+
+On startup, hirte loads configuration files from the following directories:
+
+| Load order | hirte | hirte-agent |
+|---|---|---|
+| 1 | `/usr/share/hirte/config/hirte.conf` | `/usr/share/hirte-agent/config/agent.conf` |
+| 2 | `/etc/hirte/hirte.conf` | `/etc/hirte/agent.conf` |
+| 3 | `/etc/hirte/hirte.conf.d` | `/etc/hirte/agent.conf.d` |
+
+Based on the load order, settings from a previously read configuration file will be overridden by subsequent files.
+For example, the default setting for `AllowedNodeNames` in `/usr/share/hirte/config/hirte.conf` is an empty list and
+needs to be overridden by either `/etc/hirte/hirte.conf` or files in `/etc/hirte/hirte.conf.d`. Configuration files are
```

Comment:
```suggestion
can be overridden by either editing `/etc/hirte/hirte.conf` or adding a file in `/etc/hirte/hirte.conf.d`. Configuration files are
```

Just to be sure, if I configure `AllowedNodeNames` in different files in `hirte.conf.d`, only the last one is taken into account, right? It's not combining the different values found, right?

Reply:
Yes, that is correct. There were talks about merging, but nothing more.

Reply:
Changed.